sklearn/model_selection/_search.py

```diff
@@ -1411,7 +1411,24 @@ class RandomizedSearchCV(BaseSearchCV):
         A generator over parameter settings, constructed from
         param_distributions.
 
+
+    Examples
+    --------
+    >>> from sklearn import linear_model, datasets
+    >>> from sklearn.model_selection import RandomizedSearchCV
+    >>> from scipy.stats import  uniform
```

Comment:
remove one space after `import`
```suggestion
    >>> from scipy.stats import uniform
```

Reply:
Actually, you need to fix the random state for this one.
You should replace it by:

```python
rng = np.random.RandomState(0)
...
hyperparameters = dict(C=rng.uniform(loc=0, scale=4),
...
```

Like this you are sure that you sample always the same C everytime that you are executing the example.